plugin.rb

```diff
@@ -77,6 +77,7 @@ class Engine < ::Rails::Engine
     AUTO_CLOSE_TOPIC_TIMER_CUSTOM_FIELD = "solved_auto_close_topic_timer_id"
     ACCEPTED_ANSWER_POST_ID_CUSTOM_FIELD = "accepted_answer_post_id"
     ENABLE_ACCEPTED_ANSWERS_CUSTOM_FIELD = "enable_accepted_answers"
+    IS_ACCEPTED_ASNWER_CUSTOM_FIELD = "is_accepted_answer"
```

Comment:
Added this constant so we don't keep referencing the `"is_accepted_answer"` string everywhere.